File: atcoder.jp/contests/soundhound2018-summer-qual/tasks/soundhound2018_summer_qual_b/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  string S;
  int w;
  cin >> S >> w;

  string ans = "";
  for (int i = 0; i < S.size(); i += w) {
    ans += S[i];
  }

  cout << ans << endl;
}

File: atcoder.jp/contests/soundhound2018-summer-qual/tasks/soundhound2018_summer_qual_b/main.go
package main

import (
	"fmt"
)

func main() {
	var S string
	fmt.Scanf("%s", &S)
	var w int
	fmt.Scanf("%d", &w)

	var ans []byte
	for i := 0; i < len(S); i += w {
		ans = append(ans, S[i])
	}

	fmt.Println(string(ans))
}
